### PR TITLE
Handle hub bad request issue

### PR DESCRIPTION
**Pull request by grogou**
### Description of the Change
Related to #488 
Removed try/catch case and added a new condition to check we receive `remote_id` or not

### Alternate Designs
n/a

### Benefits
We can handle the case, were not distributed the post ( really not distributed but shown like distributed in the hub ) with `post_id = 0` can't distribute because spoke is down or we don't receive some remote post id or we have a bad request.

### Possible Drawbacks
We don't have drawbacks, because we fix a bug.

### Verification Process

I have created an environment for test, downed spoke site and hub for distribution, My code tested up and work clean. This case I see in my live sites where my spoke site go down some time.

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/10up/distributor/blob/develop/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by jeffpaul** [comment]
@eriktad thanks for the PR (and related issue), we'll get this through review and back to you with any questions or concerns.

File: includes/classes/ExternalConnections/WordPressExternalConnection.php [hunk 1]
```diff
@@ -647,12 +647,12 @@ public function push( $post_id, $args = array() ) {
 
 		$body_array = json_decode( $body, true );
 
-		try {
-			$remote_id = $body_array['id'];
-		} catch ( \Exception $e ) {
+		if( !isset( $body_array['id'] ) ){
```
Comment (dinhtungdu) [inline includes/classes/ExternalConnections/WordPressExternalConnection.php:650]: we have some cs issues here (spacing)

**Comment by jeffpaul** [comment]
@eriktad can you confirm your steps to reproduce this issue as we've been unable to reproduce it?  I'm moving this to `Future Release` until we're able to reproduce the issue and if so then determine a resolution.  I agree that if we are able to reproduce this that its something we should address, its just not something we've been able to reproduce as of yet.

**Comment by arsendovlatyan** [comment]
@eriktad  Can you help @jeffpaul and his team to reproduce this issue?

I guess if we will just add "deny from all" in .htaccess of destination website, which will trigger 403 (permission denied) issue and try to push a post already existing external connection, we won't get "Could not determine remote post id." error message, which is the expected behavior.  Can you confirm this ?

**Pull request closed without merging**